stable_baselines3/common/logger.py

```diff
@@ -389,6 +414,13 @@ def write(self, key_values: Dict[str, Any], key_excluded: Dict[str, Union[str, T
             if isinstance(value, Image):
                 self.writer.add_image(key, value.image, step, dataformats=value.dataformats)
 
+            if isinstance(value, HParam):
+                # we don't use `self.writer.add_hparams` to have control over the log_dir
+                exp, ssi, sei = hparams(value.hparam_dict, metric_dict=value.metric_dict)
```

Comment:
please use explicit names, what is ssi? sei?

Reply:
It's done:
```python
experiment, session_start_info, session_end_info = hparams(value.hparam_dict, metric_dict=value.metric_dict)
```

The content and meaning of those variables is described in [the function's docstring](https://github.com/pytorch/pytorch/blob/cf5330977d10a3585358fb02c049939bf1401074/torch/utils/tensorboard/summary.py#L79-L82). (And the initial short names where those used in [pytorch SummaryWriter class](https://github.com/pytorch/pytorch/blob/master/torch/utils/tensorboard/writer.py#L336))